src/script/api/script_window.hpp

```diff
@@ -1000,6 +1000,26 @@ class ScriptWindow : public ScriptObject {
 		WID_BV_SHOW_HIDE                             = ::WID_BV_SHOW_HIDE,                             ///< Button to hide or show the selected engine.
 		WID_BV_BUILD_SEL                             = ::WID_BV_BUILD_SEL,                             ///< Build button.
 		WID_BV_RENAME                                = ::WID_BV_RENAME,                                ///< Rename button.
+		// build vehicle smart:
```

Comment:
Whole-line comments are usually marked with /* */, the simple double-slash style is reserved for end-of-line comments with very short explanations.